stow.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: stow
+  version: 2.4.1
+  epoch: 0
+  description: "GNU stow"
+  copyright:
+    - license: GPL-3.0
```

Comment:
Just a comment that the license is `GPL-3.0-or-later`, but that can be fixed in a subsequent commit.